### ISSUE TITLE
Unexpected Error has occurred popup during installation

### ISSUE THREAD

**Issue by inmento** [PeterTh/gedosato#408]
Everytime I try installing gedosato, and this is my first time ever installing this program, I get the below error. 

I have reinstalled all Microsoft runtimes several times and I didn't need to update my netframework, I was already on 4.5.

I can't get it to install, sadly.

Any help would be appreciated

See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.IO.IOException: An attempt was made to move the file pointer before the beginning of the file.

   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.SeekCore(Int64 offset, SeekOrigin origin)
   at System.IO.FileStream.Seek(Int64 offset, SeekOrigin origin)
   at System.IO.Compression.ZipArchive.ReadEndOfCentralDirectory()

************** Loaded Assemblies **************
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2633.0 built by: NET471REL1LAST_C
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework/v4.0.30319/mscorlib.dll
----------------------------------------
GeDoSaToUpdater
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/GeDoSaTo/GeDoSaToUpdater.exe
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2556.0 built by: NET471REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2556.0 built by: NET471REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2556.0 built by: NET471REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2556.0 built by: NET471REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2633.0 built by: NET471REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2612.0 built by: NET471REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
Accessibility
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2556.0 built by: NET471REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/Accessibility/v4.0_4.0.0.0__b03f5f7f11d50a3a/Accessibility.dll
----------------------------------------
System.IO.Compression.FileSystem
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2556.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.IO.Compression.FileSystem/v4.0_4.0.0.0__b77a5c561934e089/System.IO.Compression.FileSystem.dll
----------------------------------------
System.IO.Compression
    Assembly Version: 4.0.0.0
    Win32 Version: 4.7.2556.0
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.IO.Compression/v4.0_4.0.0.0__b77a5c561934e089/System.IO.Compression.dll
----------------------------------------

************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example:

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.

**Comment by chaingun427** [comment]
I've got the same problem, both on win10 and a fresh install of win7

**Comment by fiefdx** [comment]
I've got the same problem, on win10 64bit

**Comment by JohnHunt10** [comment]
Same

**Comment by ianbollinger** [comment]
Luckily the `pack` subdirectory of this repository seems to contain all the build artifacts; so I just cloned the repository. The .NET-based installer didn't work for me either, but I'd rather just replace it than put effort into diagnosing the issue.

**Comment by JohnHunt10** [comment]
How can you clone the pack repository without downloading each individual file?

**Comment by JohnHunt10** [comment]
Thanks, will try the pack folder

**Comment by JohnHunt10** [comment]
Holy, it works like a charm, guys get the pack folder like ianbollinger suggested and place it into your C: and just hit the tool.exe. Installer is broken.

**Comment by VolseniMack** [comment]
Where's the pack folder? The way ianbollinger said it, sounds like its from the "Loaded Assemblies" posted on this page.

**Comment by Torin68** [comment]
@PeterTh This appears to be a problem with .Net 4.x TLS default being SSL3 or TLS1.0 but github.com requiring TLS1.2
https://blog.github.com/2018-02-23-weak-cryptographic-standards-removed/

CJMinecraft01 had the same problem/symptoms on a project:
https://github.com/CJMinecraft01/ForgeModBuilder/issues/12

and fixed it with the following commit:
https://github.com/CJMinecraft01/ForgeModBuilder/commit/c64ab99b7b254b562e46c25561d1f56184f0d48a

Googling "ServicePointManager.SecurityProtocol" led me to the following link:
https://medium.com/@kyle.gagnet/your-net-code-could-stop-working-in-june-afb35fbf29ca

where the author gives the code fix used by CJMinecraft01 as noted above.
It also gives a link to a registry mod to enable TLS1.2 for .Net 4.x by default:
https://johnlouros.com/blog/enabling-strong-cryptography-for-all-dot-net-applications

After applying the above registry mod, GeDoSaToUpdater ran successfully on my system (Win7-x64 with .Net 4.7.1)

**Comment by mirh** [comment]
TL;DR, from an administrator command prompt:
`REG ADD "HKLM\SOFTWARE\Microsoft\.NETFramework\v4.0.30319" /v SchUseStrongCrypto /t REG_DWORD /d 1 /reg:32`

**Comment by opsynth** [comment]
> TL;DR, from an administrator command prompt:
> `REG ADD "HKLM\SOFTWARE\Microsoft\.NETFramework\v4.0.30319" /v SchUseStrongCrypto /t REG_DWORD /d 1 /reg:32`

i made an account just to say thank you. god bless people like you.
